### PR TITLE
RD2: Create Recurring Donation Schedule SObject

### PR DESCRIPTION
**Pull request by force2b**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Features Intended for Future Release
- Custom Object: RecurringDonationSchedule__c

# Deleted Metadata

### CODE REVIEW

**Comment by RadSFDeveloper** [comment]
**lurch: attach W-036547

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001CBa3QAG" target="_blank">W-036547</a>

File: src/objects/RecurringDonationSchedule__c.object [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+        <inlineHelpText>The starting date for this schedule record.</inlineHelpText>
+        <label>Start Date</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <label>Recurring Schedule</label>
+    <nameField>
+        <displayFormat>RD-Schedule-{0000000000}</displayFormat>
+        <label>Recurring Schedule Num</label>
```
Comment (RadSFDeveloper) [inline src/objects/RecurringDonationSchedule__c.object]: Can we use `Recurring Schedule Name`, `Schedule Name` or just `Name` for the record Name label?

File: src/objects/RecurringDonationSchedule__c.object [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>View</actionName>
+        <formFactor>Small</formFactor>
+        <type>Default</type>
+    </actionOverrides>
+    <allowInChatterGroups>false</allowInChatterGroups>
+    <compactLayoutAssignment>SYSTEM</compactLayoutAssignment>
+    <deploymentStatus>Deployed</deploymentStatus>
+    <description>Contains the current and past schedule(s) for each Recurring Donation. Records in this object are system maintained and should never be edited directly.</description>
```
Comment (RadSFDeveloper) [inline src/objects/RecurringDonationSchedule__c.object]: @sam-knox , can you please review Description and Help Text for the new SObject?

**Comment by force2b** [comment]
@sam-knox Thanks for the feedback.

1. No real reason for the object label other than I pulled it from the Quip doc. Recurring Donation Schedule is fine. I'll update the Quip doc with the new label as well.

1. I missed DayOfMonth. Will add.

1. I should point out that as a Protected object, this layout exists only for our internal use in Scratch orgs. Customers will never be able to open this object up.

1. I'll change the Related Campaign help text here and in the Quip doc. Also, if the Campaign on the RD is changed, then a new Schedule record will be created with the new Campaign. Generally we will not update Schedule records, only create new ones.

**Comment by Julian88Tex** [comment]
QA passed for WI/PR: https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000001CBa3QAG/view

**Pull request merged**